src/compas_viewer/components/component_manager.py

```diff
@@ -57,7 +57,7 @@ def __init__(self):
         self.all_widgets: dict = None
         self.manager_widgets = {}
 
-    def setup_manager(self):
+    def lazy_init(self):
         self.setting_components.initialize_settings()
```

Comment:
follow the naming logic, should we also change `initialize_settings()` to `lay_init()`